python/pyiceberg/utils/datetime.py

```diff
@@ -133,28 +133,24 @@ def to_human_timestamp(timestamp_micros: int) -> str:
     return (EPOCH_TIMESTAMP + timedelta(microseconds=timestamp_micros)).isoformat()
 
 
-def micros_to_hours(timestamp: int) -> int:
-    """Converts a timestamp in microseconds to a date in hours"""
-    return int((datetime.utcfromtimestamp(timestamp // 1_000_000) - EPOCH_TIMESTAMP).total_seconds() / 3600)
+def micros_to_hours(micros: int) -> int:
+    """Converts a timestamp in microseconds to hours from 1970-01-01T00:00"""
+    return micros // 3_600_000_000
```

Comment:
This converts to a timestamp only to get back to a delta, and then get the seconds. Instead, we can convert directly.